### DISCUSSION TITLE
조직 문화와 생산성

### DISCUSSION THREAD

**Discussion by TaeHyoungKwon** [ThinkAboutSoftware/AcademicConference#183] [General]
저희 회사에서 최근에 선택적 근로제를 본격 도입하면서, 결론적으로 하루에 최소 4시간은 채우면서, 주 38시간을 내가 원하는 시간에 근무 할 수 있도록 되었습니다.

이 제도를 시행하기전에 회사에서 사원들에게 다양한 의견을 받았는데요 

저같은 경우는 생산성의 측면에서, 최대한 업무하는 사람의 자율을 보장하는 쪽으로 제안을 드렸습니다.(미팅에 대한 불안감으로 필요 없이 core time을 길게 잡지 말자(결국 저희 스쿼드에서는 core time이 없어졌습니다), 개인이 자율적으로 본인의 일을 계획하고 그 일에 책임 질 수 있도록 하자 등)

마침 제가 최근에 읽던 책 중에 [수평 조직의 구조](https://www.aladin.co.kr/shop/wproduct.aspx?ItemId=231789212) 라는 책에서, 저의 제안에 대한 논거로 쓸만한 부분을 찾게 되었고, 수평조직에 대한 고민이 있으신분들에게는 좋은 내용이라 생각되어서 공유 드려 봅니다. (책 일부를 캡쳐해서 올려봅니다 - ~~혹시 문제가 된다면 후에 삭제 하겠습니다~~ 🙇  )

---------
캡쳐 내용이 4장쯤 되었는데.. 저작권법 위반일 수 있겠다 싶어서.. 후에 일부 문장 인용으로 대체하겠습니다 🙇 (캡쳐해서 올린것은 일단 삭제했습니다)

**Comment by wooyaggo86** [comment]
좋은 글귀 공유해 주셔서 감사합니다!

**Comment by jongfeel** [comment]
오 이상적인 업무 시간으로 향해 가고 있군요.

저희 회사는 06:00 ~ 22:00 사이 선택적으로 8시간 일 하는 정도인데요.
Core time이라고 해야 미팅 시간 맞추는 정도라, 사실상 자율 근무긴 하죠.

그런데 저 06:00 ~ 22:00은 사실 제가 제안한 시간인데요.
저번 회사에서도 자율 근무제여서 출퇴근 시간 간섭 안하는 회사였거든요.
그런데 그걸 악용해서 새벽에 잔뜩 일하고 낮에 자는 이상한 개발자가 
자기는 혼자 새벽에 코딩해야 집중이 잘 된다는 주장으로 그렇게 일하다 좋은 소리는 못들었거든요.

달성 가능한 목표를 정해 놓았다면, 그리고 그 기준이 동료들과 합의가 된 기준이라면
자율적으로 근무하는건 저도 동의합니다!

---

TMI) 사실 수정해서 삭제해도 commit history 마냥 남긴 합니다. 뭐 그런거 까지 확인해 가면서 보는 사람은 거의 없겠지만요.ㅎㅎ
